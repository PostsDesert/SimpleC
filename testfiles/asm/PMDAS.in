int main() {
  print(5-4);
  print(5+4);
  print(10/2);
  print(10*2);
  print(3+(4/2)*3);
  return 1;
}