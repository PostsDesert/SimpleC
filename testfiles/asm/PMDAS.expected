1
9
5
20
9
